quantized_mesh_encoder/encode.py

```diff
@@ -83,9 +99,14 @@ def encode(
 
     write_edge_indices(f, positions, n_vertices)
 
+    for ext in extensions:
+        if isinstance(ext, VertexNormalsExtension):
```

Comment:
We want this to apply to any extension
```suggestion
        if isinstance(ext, ExtensionBase):
```